src/main/kotlin/org/rust/lang/core/stubs/index/RsFeatureIndex.kt

```diff
@@ -28,8 +28,11 @@ class RsFeatureIndex : StringStubIndexExtension<RsInnerAttr>() {
         fun index(stub: RsInnerAttrStub, sink: IndexSink) {
             val metaItem = stub.psi.metaItem
             if (metaItem.name == "feature") {
-                val featureName = metaItem.metaItemArgs?.metaItemList?.singleOrNull()?.name ?: return
-                sink.occurrence(KEY, featureName)
+                val features = metaItem.metaItemArgs?.metaItemList.orEmpty()
+                for (feature in features) {
+                    val featureName = feature.name ?: continue
+                    sink.occurrence(KEY, featureName)
```

Comment:
Hm, looks like we need to bump `org.rust.lang.core.stubs.RsFileStub.Type#getStubVersion` to launch reindexing and avoid old behaviour with new plugin version

Reply:
Fixed.

Reply:
We also can bump version for this index only:
```
override fun getVersion(): Int = RsFileStub.Type.stubVersion + 1
```

Reply:
@vlad20012 Looks like it improves reindexing only for users with nightly plugin because `RsFileStub.Type.stubVersion` was already increased after prev release.
So, is it really worth it?

Reply:
@Undin ok, makes sense